extensions/plugins/system/redcore/redcore.php

```diff
@@ -119,6 +119,8 @@ public function onAfterInitialise()
 							'absoluteHrefs'     => $input->get->getBool('absoluteHrefs', true),
 						);
 
+						$this->cleanCache();
```

Comment:
maybe this option is better suited in the API itself....
and there you can use HTTP headers to get this option through too....